content/vendor/cli/tokens.md

```diff
@@ -17,11 +17,15 @@ Tokens may only access a portion of the Vendor API. The following administrative
 
 Tokens are primarily used with [Vendor CLI](https://help.replicated.com/api/replicated-vendor-cli/) or [REST API](https://help.replicated.com/api/vendor-api/) commands for automated customer, channel, and release management.
 
-## Team Tokens
+## Service Accounts
 
-Team tokens are available to all members of the vendor team. These tokens can be created, retrieved, and revoked by any user with the proper RBAC policy.
+Service accounts are assigned a token and associated with an RBAC policy. Any user with the proper permissions can create, retrieve or revoke them.
 
-Team tokens can be set to Read Only or Read / Write privileges.
+Service account names must be unique within a given team. Admin users can assign any RBAC policy to a service account. Non-admin users can only assign their own RBAC policy.
+
+Service account tokens are only displayed once when created. The token may never be retrieved again after initial creation. It is highly recommended to store the token in a safe place, such as a password vault.
```

Comment:
there's a lot of repetition here and i'm wondering if it makes sense to pull some of the common features (like displaying only once, etc) into the section above. i don't feel strongly about it though.

Reply:
I agree but also couldn't come up with a great suggestion since there is a lot of overlap with service/user tokens, but some specifics that apply to one and not the other. Since it isn't a super long section though and no good alternatives it seemed that the repetition would okay for now

Reply:
yeah i think it's fine for now